### PR TITLE
Add chrono to use for new todo item

### PR DESCRIPTION
**Pull request by amahalwy**

### CODE REVIEW

File: components/AddTodo.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import * as chrono from "chrono-node";
+import { Form, Field } from "react-final-form";
+
+const AddTodo = ({ todos, setTodos }) => {
+  const required = (value) => (value ? undefined : "Required");
+  const onSubmit = (values) => {
+    values.status = "Not Started";
+    values.active = false;
+    const res: any = chrono.parseDate(values.description);
+    const newD: any = new Date();
```
Comment (mmahalwy) [inline components/AddTodo.tsx:19]: the type here is `Date` not `any`

File: components/AddTodo.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+} from "@chakra-ui/react";
+import * as chrono from "chrono-node";
+import { Form, Field } from "react-final-form";
+
+const AddTodo = ({ todos, setTodos }) => {
+  const required = (value) => (value ? undefined : "Required");
+  const onSubmit = (values) => {
+    values.status = "Not Started";
+    values.active = false;
+    const res: any = chrono.parseDate(values.description);
```
Comment (mmahalwy) [inline components/AddTodo.tsx:18]: whats the type here?

File: components/DisplayInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { Button, Box, Text } from "@chakra-ui/react";
+import { GiPauseButton } from "react-icons/gi";
+import { BsPlayFill } from "react-icons/bs";
+import { DisplayInfoProps } from "../typescript/interfaces";
+
+const timeConvert = (num) => {
```
Comment (mmahalwy) [inline components/DisplayInfo.tsx:7]: this needs to be prettier. having `119:30 mins` is super ugly? I think there are libs that help you with this

File: components/DisplayInfo.tsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  status,
+  timeLeft,
+  activeTodo,
+  toggleActive,
+}) => {
+  if (!todo.duration) return null;
+  if (status === "Not Started") {
+    return (
+      <Button onClick={start} disabled={activeTodo && activeTodo !== todo.id}>
+        Start?
```
Comment (mmahalwy) [inline components/DisplayInfo.tsx:34]: remove `?`

File: components/AddTodo.tsx (hunk 1 above)
Comment (amahalwy) [reply components/AddTodo.tsx:19]: Yes, but to be able to do `values.duration = res > 0 ? ((res - newD) / 60000).toFixed(2) : null;` an error of arithmetics occurs that states newD must either be a `number` or `any` type

File: components/AddTodo.tsx (hunk 2 above)
Comment (amahalwy) [reply components/AddTodo.tsx:18]: On `res` or values.status/active?

**Pull request merged**